.github/workflows/publish.yml

```diff
@@ -9,8 +9,10 @@ jobs:
   publish-rockspec-scm-1:
     if: github.ref == 'refs/heads/master'
     runs-on: ubuntu-latest
+    needs: version_check
```

Comment:
As far as I remember GitHub actions, you can't depend on another workflow job like this. For now it hadn't failed because this workflow doesn't run on PRs.

https://stackoverflow.com/questions/58457140/dependencies-between-workflows-on-github-actions

You need to create another job here. The same as in packaging. A bit of copypaste, yeah, but since it's like three lines of code (after you migrate checkout), it shouldn't be an issue.

Reply:
Done, implemented job `version-check` in publish.yml

> A bit of copypaste, yeah, but since it's like three lines of code (after you migrate checkout), it shouldn't be an issue.

Related [with](https://github.com/tarantool/queue/pull/199#discussion_r1126262069)